app/views/public/api_documentation.html.erb

```diff
@@ -394,6 +395,63 @@
 
 <hr />
 
+<h2 class="h4" id="stake-explorer">Stake Explorer</h2>
+<p>
+  The Stake Accounts endpoint will return all the stake accounts according to provided criteria. Only mainnet network
+  is available for this query.
```

Comment:
I'd add that parameter is required - see available options below, or something along those lines